### PR TITLE
SQDEVOPS-309 update nginx dependency

### PR DESCRIPTION
**Pull request by jCOTINEAU**

### CODE REVIEW

File: charts/sonarqube-dce/Chart.yaml [hunk 1]
```diff
@@ -70,6 +70,6 @@ dependencies:
     repository: https://raw.githubusercontent.com/bitnami/charts/pre-2022/bitnami
     condition: postgresql.enabled
   - name: ingress-nginx
-    version: 4.7.1
+    version: 4.8.3
```
Comment (carminevassallo) [inline charts/sonarqube-dce/Chart.yaml:73]: Hey @jCOTINEAU,

shall we log this update in the change list? I think we haven't always done this in the past, but we should. WDYT?

**Comment by carminevassallo** [comment]
A minor comment on the commit message. Shall we adopt the following structure for this and future dependency updates? "Update X to version Y"? If yes, we might add a note in the helm dev guidelines...

File: charts/sonarqube-dce/Chart.yaml (hunk 1 above)
Comment (jCOTINEAU) [reply charts/sonarqube-dce/Chart.yaml:73]: that is a very good catch doing it now

**Review by carminevassallo — APPROVED** [review]
LGTM!

**Pull request merged**